features/topology-map/plugins/org.opennms.features.topology.plugins.topo.graphml/src/main/java/org/opennms/features/topology/plugins/topo/graphml/GraphMLTopologyProvider.java

```diff
@@ -120,6 +107,26 @@ public GraphMLTopologyProvider(GraphMLGraph graph, GraphMLServiceAccessor servic
         }
     }
 
+    private void setNodeIdForVertex(GraphMLVertex vertex) {
+        if (Objects.isNull(vertex)) {
```

Comment:
Is there a particular reason to use `Objects.isNull()` instead of just verifiing if `vertex == null`?